pages/phases/resume-review/index.html

```diff
@@ -1,11 +1,11 @@
 ---
 layout: article
-permalink: /hiring-phases/reviewing-resumes/
+permalink: /hiring-phases/resume-review/
 section: hiring-phases
 sidenav: hiring-phases
-phase: reviewing-resumes
-title: Reviewing Resumes
-intro: Before SMEs begin reviewing the resumes, remove applicants who are ineligible for federal employment or who did not include a resume with their application. SMEs will independently review each remaining applicant’s resume to determine whether they adequately reflect the core competencies and proficiencies to warrant the first of two phone assessment interviews.
+phase: resume-review
+title: Resume Review
+intro: SMEs independently review each remaining applicant’s resume to determine whether they adequately reflect the core competencies and proficiencies to warrant entry to the assessment phase.
 roles: [hr, sme]
 duration: Approximately 4 days
```

Comment:
i think we say 5 days for this not 4

Reply:
Done: https://github.com/usds/SME-QA-Website/pull/243/commits/07f713a166a55a750c7e48edddd22a60ea1d0d0b